### ISSUE TITLE
Lesson 1.2 typo

### ISSUE THREAD

**Issue by softwaremike** [petabridge/akka-bootcamp#79]
This is the code for "GetAndValidateInput" from the lesson 1.2 email:
```c#
/// <summary>
/// Reads input from console, validates it, then signals appropriate response
/// (continue processing, error, success, etc.).
/// </summary>
private void GetAndValidateInput()
{
    var message = Console.ReadLine();

...

    else
    {
        var valid = IsValid(message);
        if (valid)
        {
            _consoleWriterActor.Tell(new Messages.InputSuccess("Thank you! Message was valid."));
        }
        else
        {
            Self.Tell(new Messages.ValidationError("Invalid: input had odd number of characters."));
        }
     
        // continue reading messages from console
        Self.Tell(new Messages.ContinueProcessing());
    }
}
```

The "Self.Tell" in this line should actually be _ConsoleWriterActor.Tell:
```c#
    Self.Tell(new Messages.ValidationError("Invalid: input had odd number of characters."));
```

**Comment by skotzko** [comment]
At this point in the lessons we're still showing the non-`ReceiveActor` pattern matching, so what's happening here is that the `ConsoleReaderActor` is sending itself a specific instance of the `ValidationError` message, which then gets matched and sent on to the `ConsoleWriterActor` in a more generic form  ([see here](https://github.com/petabridge/akka-bootcamp/blob/master/src/Unit-1/lesson2/Completed/ConsoleReaderActor.cs#L27)).

While we wrote it this way intentionally, it could go either way and just happened to be the way we wrote it here.

**Comment by softwaremike** [comment]
This change I suggest also fixed the double console.readline() problem that was fixed in a different way in commit 67be47c.  Also, from an outsider's perspective, the way it's written currently just "looks" wrong, and seems to be a needlessly complex way to show an error - ie, send myself an "error" message; then, when receiving the message, detect if it's an error message, and if so, forward on to another actor.  Anyway, just my two cents, and no big deal either way; and I appreciate the effort put into these lessons!  Very nice!
